diff --git a/corners/theme.conf b/corners/theme.conf
index 756f2e2..0b6ed2f 100644
--- a/corners/theme.conf
+++ b/corners/theme.conf
@@ -18,8 +18,8 @@
 # Scale:        number. Adjusts the size of UI elements. Can be anything, but I
 #               would stick between 1 and 2 (e.g. 1.5).
 
-BgSource="backgrounds/glacier.png"
-FontFamily="Atkinson Hyperlegible"
+BgSource="backgrounds/leaves.png"
+FontFamily="CaskaydiaMono Nerd Font"
 FontSize=9
 Padding=50
 Radius=10
@@ -126,4 +126,4 @@ TimeColor="#cdd6f4"
 TimeSize=48
 TimeIsBold=true
 TimeOpacity=1.0
-TimeFormat="hh:mm AP"
+TimeFormat="HH:mm"
